examples/Secrets/gitops.json

```diff
@@ -429,8 +460,7 @@
     "index": {
       "introduction": {
         "heading": "Open Access Data Commons",
-        "text": "The Open Access Data Commons supports the management, analysis and sharing of data for the research community with the aim of accelerating discovery and development of diagnostics, treatment and prevention of diseases.",
-        "link": "/submission"
+        "text": "The Open Access Data Commons supports the management, analysis and sharing of data for the research community with the aim of accelerating discovery and development of diagnostics, treatment and prevention of diseases."
```

Comment:
Gitops Updated. In the "login" section of the gitops there is a place to put contact,email,and image information. 

I copy pasted the sentence from the home screen to the login since stock gen3 did the same but this should be further considered

NOTE: I do not think the footer can be removed in gitops. This could be done in a local build of windmill.

The current Front page And login screens: 

<img width="753" alt="Screen Shot 2023-02-13 at 9 15 55 AM" src="https://user-images.githubusercontent.com/33436238/218526432-0b8d3a4a-5327-4134-ae2f-d91545d1b353.png">

<img width="729" alt="Screen Shot 2023-02-13 at 9 17 45 AM" src="https://user-images.githubusercontent.com/33436238/218526730-ffe363d2-ce52-49ca-9171-cd723b835619.png">